### DISCUSSION TITLE
How to use the onboard “animate” library for Astroid elements (e.g. Sticky Header Logo ) ?

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#289] [Q&A]
**How to use the onboard “animate” library for Astroid elements?**

In this case the “Sticky Header Logo”

I try to add a class to the element using “**Animate**” class, but it don't seem to work?
https://animate.style/

**Comment by sonvnn** [comment]
Hi @Chacapamac 

You can add this code in Custom Code tab
```
<link
    rel="stylesheet"
    href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"
  />
```

And now you can add their effect via classes to layout builder or anywhere as you want.

Best Regards,
Sonny

**Reply by Chacapamac** [reply]
@sonvnn 
I ask because you already have the **animate.min.css** here:
![Screen Shot 2023-03-21 at 2 47 52 PM](https://user-images.githubusercontent.com/1907276/226712083-bc7dacb2-199c-4f3d-bda2-cf669ccc4234.jpg)

**animate.min.css** seems to not be loaded by Astroid, it seem to load ONLY if I use an animation in the layout and that make perfect sens.

**If I prefer to use the local **animate.min.css** than an external call (and not using any animation in the layout) can I load it using a Custom CSS Files link?**

**Reply by Chacapamac** [reply]
@sonvnn 

animate.min.css seems to not be loaded by Astroid, it seem to load ONLY if I use an animation in the layout and that make perfect sens.

If I prefer to use the local animate.min.css than an external call (and not using any animation in the layout) can I load it using a Custom CSS Files link?

**Reply by Chacapamac** [reply]
**Well I try everything....**
Logically adding the Joomla Root —> _**media/astroid/assets/css/animate.min.css**_ should be the same then adding it by the cloud, but it doesn't seem to work???

When using the Astroid Template Manager to add an animation, Astroid load something like:
<link href="http://yourwebsite.com/media/astroid/assets/css/animate.min.css?3401bcbc44dcab5d56ab4b99b9b8829a" rel="stylesheet" type="text/css">

I will come back with a complete solution that not load an external cloud file or use the animate.min.css file already in Astroid system.
